frontend/src/scenes/feature-flags/featureFlagLogic.test.ts

```diff
@@ -0,0 +1,142 @@
+import { expectLogic, partial } from 'kea-test-utils'
+import { MOCK_DEFAULT_PROJECT } from 'lib/api.mock'
+
+import { useMocks } from '~/mocks/jest'
+import { initKeaTests } from '~/test/init'
+import { FeatureFlagType, PropertyFilterType, PropertyOperator } from '~/types'
+
+import { featureFlagLogic, NEW_FLAG } from './featureFlagLogic'
+
+const MOCK_FEATURE_FLAG = {
+    ...NEW_FLAG,
+    id: 1,
+    key: 'test-flag',
+    name: 'test-name',
+}
+
+const MOCK_FEATURE_FLAG_STATUS = {
+    status: 'active',
+    reason: 'mock reason',
+}
+
+describe('featureFlagLogic', () => {
+    let logic: ReturnType<typeof featureFlagLogic.build>
+
+    beforeEach(async () => {
+        initKeaTests()
+        logic = featureFlagLogic({ id: 1 })
+        logic.mount()
+
+        useMocks({
+            get: {
+                [`/api/projects/${MOCK_DEFAULT_PROJECT.id}/feature_flags/${MOCK_FEATURE_FLAG.id}/`]: () => [
+                    200,
+                    MOCK_FEATURE_FLAG,
+                ],
+                [`/api/projects/${MOCK_DEFAULT_PROJECT.id}/feature_flags/${MOCK_FEATURE_FLAG.id}/status`]: () => [
+                    200,
+                    MOCK_FEATURE_FLAG_STATUS,
+                ],
+            },
+        })
+
+        await expectLogic(logic).toFinishAllListeners()
+    })
+
```

Comment:
Lets unmount after each.

```suggestion
    afterEach(() => {
        logic.unmount()
    })

```